### DISCUSSION TITLE
force JIT to compile all themes

### DISCUSSION THREAD

**Discussion by michabbb** [tailwindlabs/tailwindcss#7726] [Help] [answered]
hi,

i am asking in the context of a typical laravel mix setup, like:

```js
mix.postCss('resources/css/app.css', 'public/css', [
        require('postcss-import'),
        require('tailwindcss'),
    ]);
```

```js
module.exports = {
    content: [
        './vendor/laravel/jetstream/**/*.blade.php',
        './storage/framework/views/*.php',
        './resources/views/**/*.blade.php',
    ],
```

when using `JIT` only styles get compiled, that are anywhere used inside any blade file, which I do understand 😏 
but in my special case, I am adding "theme-classes" dynamically to the `<body>` based on the URL.
and the class-name comes from the code - so the JIT compiler does not find these theme-classes and so they don´t get generated.
I do understand this as well.

so I created a "fake" blade file like:

```html
<div class="theme1 theme2 theme3 theme4"></div>
```

That file is never used but with that file, JIT is now able to find my theme-classes and compiles them.
I can live with that solution, but I wanted to ask if there is any method to tell the JIT compiler to compile something that it does not find on its own ?

something like:

```js
module.exports = {
   jit: {  
     force-compile-themes: ['theme1','theme2','theme3']
   }
}
```

thanks for any help/advise!

**Comment by robbyrice** [comment, marked as answer]
I think this is what you are looking for [https://tailwindcss.com/docs/content-configuration#safelisting-classes](https://tailwindcss.com/docs/content-configuration#safelisting-classes)
